doc/en/explanation/goodpractices.rst

```diff
@@ -159,26 +124,11 @@ sub-directory of your root:
             __init__.py
             app.py
             view.py
-    tests/
-        __init__.py
-        foo/
-            __init__.py
-            test_view.py
-        bar/
-            __init__.py
-            test_view.py
-
+    tests/...
 
 This layout prevents a lot of common pitfalls and has many benefits, which are better explained in this excellent
 `blog post by Ionel Cristian Mărieș <https://blog.ionelmc.ro/2014/05/25/python-packaging/#the-structure>`_.
 
-.. note::
-    The ``--import-mode=importlib`` option (see :ref:`import-modes`) does not have
-    any of the drawbacks above because ``sys.path`` is not changed when importing
-    test modules, so users that run into this issue are strongly encouraged to try it.
-
-    The ``src`` directory layout is still strongly recommended however.
-
 
 Tests as part of application code
```

Comment:
We should remove `setup.cfg` from the listing below too (I can't comment directly on it because it is not part of the diff).